registration/include/pcl/registration/impl/ndt.hpp

```diff
@@ -427,22 +430,20 @@ NormalDistributionsTransform<PointSource, PointTarget>::computeHessian (Eigen::M
     {
       cell = *neighborhood_it;
 
-      {
-        x_pt = input_->points[idx];
-        x = Eigen::Vector3d (x_pt.x, x_pt.y, x_pt.z);
+      x_pt = input_->points[idx];
```

Comment:
`input_[idx]`

Reply:
Do you mean modifying it to like ```x_pt = input_->at(idx);```?

Reply:
Apologies. `(*input_)[idx]`. The goes is to prevent direct use of `PointCloud::points` because it's not needed.

Reply:
OK, I got rid of all ```PointCloud::points``` through the code.